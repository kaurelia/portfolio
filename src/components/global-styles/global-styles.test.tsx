import { render } from "@testing-library/react";
import AppProvider from "../app-provider/app-provider";
import GlobalStyles from "./global-styles";

describe("Test global-styles component", () => {
  it("Check if global-styles renders properly", async () => {
    const { container } = render(
      <AppProvider>
        <GlobalStyles />
      </AppProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
