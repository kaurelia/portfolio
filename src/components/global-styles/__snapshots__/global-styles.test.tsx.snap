// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test global-styles component Check if global-styles renders properly 1`] = `<div />`;
